test/pylib/resource_gather.py

```diff
@@ -76,30 +76,6 @@ def __del__(self):
         if self.own_loop:
             self.loop.close()
 
-    def  run_process(self, args: list[str], timeout, env: dict = None) -> [subprocess.Popen, str]:
```

Comment:
It looks like you added this method [here](https://github.com/scylladb/scylladb/pull/22894/commits/8d60ce8952a69471a6a3e3d782efc765edd1c5cb), now it's removed.